### PR TITLE
Added Operator Upgrade Tests from release/3.4 

### PR DESCRIPTION
**Pull request by anpanigr**
Added new Upgrade test to 

  (a) Upgrade Mii/Domain-In Image Domain from release/3.4
  (b) Upgrade FMW domain ( Domain on PV) from  relese/3.4   
  (c) Upgrade AuxDomain ( v8 schema) from release/3.4

https://build.weblogick8s.org:8443/job/wko-kind-nightly-seqential/655

There are two failures due to issue  [OWLS-99119]
ItOperatorWlsUpgrade/testOperatorWlsAuxDomainUpgradeFrom340ToCurrent  (new) 
ItOperatorWlsUpgrade/testOperatorWlsAuxDomainUpgradeFrom338ToCurrent  (regression)

Following new tests methods passed 
ItOperatorWlsUpgrade/testOperatorWlsUpgradeFrom340ToCurrent(_String__1)
ItOperatorWlsUpgrade/testOperatorWlsUpgradeFrom340ToCurrent(_String__2)
ItOperatorFmwUpgrade.testOperatorFmwUpgradeFrom340ToCurrent

### CODE REVIEW

**Review by xiancao — APPROVED** [review]
LGTM

File: integration-tests/src/test/resources/bash-scripts/syncTestRepo.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/up
+
+SOURCE_REPO=${BASE_IMAGES_REPO}
+SOURCE_USER=${BASE_IMAGES_REPO_USERNAME}
+SOURCE_PASSWORD=${BASE_IMAGES_REPO_PASSWORD}
```
Comment (sankarpn) [inline integration-tests/src/test/resources/bash-scripts/syncTestRepo.sh:7]: After your REPO changes PR is done, we will still reference OCR(or)OCIR for base images? If yes we should use OCR as the BASE_IMAGES_REPO here.

File: integration-tests/src/test/resources/bash-scripts/syncTestRepo.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+dockerPullPushDbImage() {
+ image=${1}
+ docker pull ${SOURCE_REPO}/database/${image}
+ docker tag ${SOURCE_REPO}/database/${image} \
+            ${TARGET_REPO}/weblogick8s/test-images/database/${image}
+ docker push  ${TARGET_REPO}/weblogick8s/test-images/database/${image}
+ docker rmi -f ${SOURCE_REPO}/database/${image}
+}
+
```
Comment (sankarpn) [inline integration-tests/src/test/resources/bash-scripts/syncTestRepo.sh:42]: Ideally we should maintain a file with a list of images with complete URI and then read it from 1 by one and pull and push it to local repo. We should not hardcode images here.

**Review by sankarpn — CHANGES_REQUESTED** [review]

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**